pkg/datapath/linux/sysctl/cell.go

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package sysctl
+
+import (
+	"github.com/spf13/afero"
+	"github.com/spf13/pflag"
+
+	"github.com/cilium/cilium/pkg/datapath/tables"
+	"github.com/cilium/cilium/pkg/hive/cell"
+	"github.com/cilium/cilium/pkg/statedb/reconciler"
+)
+
+var Cell = cell.Module(
+	"sysctl",
+	"Tweak sysctl parameters and reconcile them with the kernel",
+
+	cell.Config(Config{}),
+
+	cell.Provide(
+		newSysctl,
+	),
+	cell.ProvidePrivate(
+		tables.NewSysctlTable,
+		newReconcilerConfig,
+		newReconcilerOps,
+	),
+	cell.ProvidePrivate(
+		func() afero.Fs {
+			return afero.NewOsFs()
+		},
+	),
+	cell.Invoke(reconciler.Register[*tables.Sysctl]),
+)
+
+type Config struct {
+	ProcFs string `mapstructure:"procfs"`
+}
+
+func (cfg Config) Flags(flags *pflag.FlagSet) {
+	flags.String("procfs", "/proc", "Root's proc filesystem path")
```

Comment:
I know this is the original doc string, but it's pretty weird. Perhaps "Path to the proc filesystem" or "Proc filesystem mount point" or "Path to the host's proc filesystem mount"?